job-server/src/main/resources/reference.conf

```diff
@@ -0,0 +1,6 @@
+spark {
+  jobserver {
+    # Maximum number of dispatcher threads for each Spark context
+    max-dispatchers-per-context = 2
```

Comment:
Two comments:
1) I think the name `max-threads-per-context` is better description, at least I think of dispatchers as like ExecutionContext.
2) 2 seems really low, and means that by default a user can only have 2 jobs in parallel.  How about a default of 4, at least?